src/main/java/athena/Storage.java

```diff
@@ -40,7 +42,7 @@ public void saveTaskListData(TaskList tasks) {
     public TaskList loadTaskListData() {
         File csvFile = new File(filePath);
         TaskList output = new TaskList();
-        int maxIndex = -1;
+        int maxNumber = 0;
```

Comment:
There is a possibility of an empty save file, should initialise maxNumber to the default of -1?

Reply:
Actually I wasn't sure should the number start from 0 or 1. I thought our first task would start from 1?

Reply:
The TaskList initialises the maxNumber as -1. We can either change it here or there as long as its standardised. I suggest changing it here because I used this in some of the test code and allows us to check fo rpotentially unused TaskLists